### PR TITLE
gate/README.md  Fix link to .cirrus.yml and reword

### PR DESCRIPTION
**Pull request by abitrolly**

### CODE REVIEW

File: contrib/gate/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![PODMAN logo](../../logo/podman-logo-source.svg)
 
 A standard container image for lint-checking and validating changes to the libpod
-repository.  The
-[contributors guide contains the documentation for usage.](https://github.com/containers/libpod/blob/master/CONTRIBUTING.md#go-format-and-lint).  Note that this container image is also utilized
-in automation, see the file [.cirrus.yml](.cirrus.yml)
+repository.  It must be build from the repository root as
```
Comment (vrothberg) [inline contrib/gate/README.md:4]:
```suggestion
repository.  It must be built from the repository root as
```

Comment (abitrolly) [reply contrib/gate/README.md]: Oops. :D

File: contrib/gate/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ![PODMAN logo](../../logo/podman-logo-source.svg)
 
 A standard container image for lint-checking and validating changes to the libpod
```
Comment (TomSweeneyRedHat) [inline contrib/gate/README.md:3]: I know it's preexisting, but could you change the start of the first sentence to `The "gate" images is a standard container image ...`

Comment (abitrolly) [reply contrib/gate/README.md]: That sounds better. Thanks.

**Pull request merged**